task/src/routers/controllers/group/group.controller.ts

```diff
@@ -10,32 +10,38 @@ import {
   Put,
   Query,
   Res,
+  UseGuards,
 } from '@nestjs/common';
 import { Response } from 'express';
 import { GroupService } from 'src/services/group/group.service';
 import { Group } from 'src/data-access/entities/group.entity';
 import { GroupDTO } from 'src/interfaces/dto/groupdto';
+import { JwtAuthGuard } from 'src/guards/jwt/jwt-auth.guard';
 
 @Controller('groups')
```

Comment:
if you use the same guard to every controller method you can set it for the whole controller (before the line 21)